centreon/www/front_src/src/Resources/Listing/index.cypress.spec.tsx

```diff
@@ -0,0 +1,462 @@
+import { Column, Method, TestQueryProvider } from '@centreon/ui';
+
+import { Resource, ResourceType } from '../models';
+import Context, { ResourceContext } from '../testUtils/Context';
+import useActions from '../testUtils/useActions';
+import useFilter from '../testUtils/useFilter';
+import { labelInDowntime, labelAcknowledged } from '../translatedLabels';
+import { getListingEndpoint, defaultSecondSortCriteria } from '../testUtils';
+import { unhandledProblemsFilter } from '../Filter/models';
+import useLoadDetails from '../testUtils/useLoadDetails';
+import useDetails from '../Details/useDetails';
+import { resourcesToAcknowledgeAtom } from '../Actions/actionsAtoms';
+
+import useListing from './useListing';
+import { getColumns, defaultSelectedColumnIds } from './columns';
+
+import Listing from '.';
+
+const columns = getColumns({
+  actions: {
+    resourcesToAcknowledgeAtom
+  },
+  t: identity
+}) as Array<Column>;
+
+const fillEntities = (): Array<Resource> => {
+  const entityCount = 31;
+
+  return new Array(entityCount).fill(0).map((_, index) => ({
+    acknowledged: index % 2 === 0,
+    duration: '1m',
+    id: index,
+    in_downtime: index % 3 === 0,
+    information:
+      index % 5 === 0 ? `Entity ${index}` : `Entity ${index}\n Line ${index}`,
+    last_check: '1m',
+    links: {
+      endpoints: {
+        acknowledgement: `/monitoring/acknowledgement/${index}`,
+        details: 'endpoint',
+        downtime: `/monitoring/downtime/${index}`,
+        metrics: 'endpoint',
+        performance_graph: index % 6 === 0 ? 'endpoint' : undefined,
+        status_graph: index % 3 === 0 ? 'endpoint' : undefined,
+        timeline: 'endpoint'
+      },
+      externals: {
+        notes: {
+          url: 'https://centreon.com'
+        }
+      },
+      uris: {
+        configuration: index % 7 === 0 ? 'uri' : undefined,
+        logs: index % 4 === 0 ? 'uri' : undefined,
+        reporting: index % 3 === 0 ? 'uri' : undefined
+      }
+    },
+    name: `E${index}`,
+    passive_checks: index % 8 === 0,
+    severity_level: index % 3 === 0 ? 1 : 2,
+    short_type: index % 4 === 0 ? 's' : 'h',
+    status: {
+      name: index % 2 === 0 ? 'OK' : 'PENDING',
+      severity_code: index % 2 === 0 ? 2 : 5
+    },
+    tries: '1',
+    type: index % 4 === 0 ? ResourceType.service : ResourceType.host,
+    uuid: `${index}`
+  }));
+};
+
+const entities = fillEntities();
+const retrievedListing = {
+  meta: {
+    limit: 30,
+    page: 1,
+    search: {},
+    sort_by: {},
+    total: entities.length
+  },
+  result: entities
+};
+
+let context: ResourceContext;
+
+const ListingTest = (): JSX.Element => {
+  const listingState = useListing();
+  const actionsState = useActions();
+  const detailsState = useLoadDetails();
+  const filterState = useFilter();
+
+  useDetails();
+
+  context = {
+    ...listingState,
+    ...actionsState,
+    ...detailsState,
+    ...filterState
+  };
+
+  return (
+    <Context.Provider value={context}>
```

Comment:
I think we can try to remove everything related to the Context.
This includes to not remove the call to useDetails, useFilter, useLoadDetails. Replace their import statement in order to use the real implementation rather than the `testUtils` one